Server/KitchenDevice.java

```diff
@@ -0,0 +1,12 @@
+public class KitchenDevice extends Device {
+
+      private List<Order> order_list;
+      private List<Notification> notification_list;
+
+      public kitchenDevice(List<Order> order_list, List<Notification> notification_list) {
```

Comment:
Αρχικά το notification_list είναι σίγουρα άδειο, άρα δε χρειάζεται να το περάσουμε ως παράμετρο στον constructor. Απλά δημιουργούμε το αντικείμενο με 

`this.notification_list = new ArrayList<Notification>();`

μέσα στον constructor ή με

`private List<notification> notification_list = new ArrayList<>();`

στη δήλωση της μεταβλητής.